README.Rmd

```diff
@@ -52,3 +52,13 @@ For example, suppose we have a table to load with 2 integer columns, we could us
     c=rs_upsert_table(nx, dbcon=con, tableName = 'mytable', split_files=4, bucket="mybucket", keys=c('a'))
 
 ```
+
+## Changelog
+
+### 0.2.2
+
+* Updated code to meet {dbplyr}'s change in class from `dbplyr::in_schema()`
+* BREAKING CHANGE: standardized function call args from `tableName` to `table_name`
+* Updated code wrapped in transaction() so that errors bubble up
+* {redshiftTools} tests had been broken for a while, did some bug fixes o that they pass
+* Because we now make good use of schema other than 'public' earlier introspection code using `pg_table_def` stopped working unless we were to update the user's search path. In order to avoid that, we now use `information_schema.columns` for that.  :ht: @ Jessica Holliday.
```

Comment:
```suggestion
* Because we now make good use of schema other than 'public' earlier introspection code using `pg_table_def` stopped working unless we were to update the user's search path. In order to avoid that, we now use `information_schema.columns`.  :ht: @ Jessica Holliday.
```